redwood-runtime/src/commonMain/kotlin/app/cash/redwood/LayoutModifier.kt

```diff
@@ -46,6 +46,11 @@ public interface LayoutModifier {
    */
   public fun <R> foldOut(initial: R, operation: (Element, R) -> R): R
 
+  /**
+   * Iterates over all [Element]s in this [LayoutModifier].
+   */
+  public fun forEach(block: (Element) -> Unit)
```

Comment:
The generated serialization code can also use this. Currently it's doing a `foldIn(Unit)` to emulate a for loop.

Reply:
Updated the generated code.